Probe/src/main.go

```diff
@@ -18,30 +18,16 @@ package main
 
 import (
 	"flag"
-	"github.com/golang/protobuf/proto"
-	"log"
-	"utils"
 
-	//"github.com/golang/protobuf/proto"
-	//"log"
-	"probe"
-	//"utils"
+	"github.com/FirebaseExtended/fcm-external-prober/Probe/src/probe"
+	"github.com/FirebaseExtended/fcm-external-prober/Probe/src/utils"
 )
 
-func initFlags() (*probe.ProbeConfigs, *probe.AccountInfo) {
-	p := flag.String("probes", "", "ProbeConfigs protobuf with probe behaviors")
-	a := flag.String("account", "", "AccountInfo protobuf with GCP account info")
+func initFlags() (*string, *string) {
```

Comment:
you assign the values to cfgs and acct here, and you do it in main() as well.  Just do it once.

I think it makes sense that the unmarshalling happen here, but open to other thoughts.  My thinking is that it is better to crash as soon as possible if the config is invalid.

Reply:
addressed in 009b737